llvm/lib/Transforms/Vectorize/LoopVectorize.cpp

```diff
@@ -1248,8 +1248,10 @@ class LoopVectorizationCostModel {
   /// \return The desired interleave count.
   /// If interleave count has been specified by metadata it will be returned.
   /// Otherwise, the interleave count is computed and returned. VF and LoopCost
-  /// are the selected vectorization factor and the cost of the selected VF.
-  unsigned selectInterleaveCount(ElementCount VF, InstructionCost LoopCost);
+  /// are the selected vectorization factor and the cost of the selected VF for
```

Comment:
There should be no need to pass `L`, as `LoopVectorizationCostMOdel` stores it already in `TheLoop`?